cristin-import/src/main/resources/ignoredFields.txt

```diff
@@ -7,6 +7,7 @@ modifiedDate
 projects
 publishedDate
 doiRequest
+.owner
```

Comment:
owner cannot be empty. it is part of the primary key. inject a value when you create the test data

Reply:
There is a method in CirstinObject called hardcodePublicationOwner